### PR TITLE
Feat: kafka to postgresql

### PR DESCRIPTION
**Pull request by Ayato77**
### Description
- Option for AutoCommit.Enable
- Export Client.Producer and Clien.Consumer
- Client.saramaClient is added to create OffsetManager
- New methods for OffsetManager

### CODE REVIEW

File: pkg/kafka/client.go [hunk 1]
```diff
@@ -132,14 +135,23 @@ func NewKafkaClient(opts NewClientOptions) (client *Client, err error) {
 	zap.S().Debugf("Getting consumer for %s", opts.ConsumerName)
 	zap.S().Debugf("Brokers: %v", opts.Brokers)
 	zap.S().Debugf("Config: %#v", config)
-	client.consumer, err = sarama.NewConsumerGroup(opts.Brokers, opts.ConsumerName, config)
+
+	client.saramaClient, err = sarama.NewClient(opts.Brokers, config)
```
Comment (Scarjit) [inline pkg/kafka/client.go:139]: Replace with 
config.Consumer.Offsets.AutoCommit.Enable = opts.AutoCommit

File: pkg/kafka/client.go [hunk 2]
```diff
@@ -578,3 +590,16 @@ func NewConsumerGroupHandler(queue *chan Message) sarama.ConsumerGroupHandler {
 		queue: queue,
 	}
 }
+
+func (c *Client) CreateOffsetManager(consumerName string) (sarama.OffsetManager, error) {
```
Comment (Scarjit) [inline pkg/kafka/client.go:594]: c.saramaClient with c.consumer

File: pkg/kafka/client.go [hunk 3]
```diff
@@ -578,3 +590,16 @@ func NewConsumerGroupHandler(queue *chan Message) sarama.ConsumerGroupHandler {
...
+	offsetManager, err := sarama.NewOffsetManagerFromClient(consumerName, c.saramaClient)
+	if err != nil {
+		zap.S().Errorf("Error creating offsetmanager: %s", err)
+		return nil, err
+	}
+	return offsetManager, nil
+}
+
+func (c *Client) GetAutoCommitEnableStatus() bool {
```
Comment (Scarjit) [inline pkg/kafka/client.go:603]: c.saramaClient with c.consumer

**Review by Scarjit — CHANGES_REQUESTED** [review]

File: pkg/kafka/client.go (hunk 1 above)
Comment (Ayato77) [reply pkg/kafka/client.go:145]: Replaced.

**Review by Scarjit — APPROVED** [review]

**Pull request merged**